core/Db/Schema/Mysql.php

```diff
@@ -781,6 +786,43 @@ public function getSupportedReadIsolationTransactionLevel(): string
         return 'READ UNCOMMITTED';
     }
 
+    public function hasReachedEOL(): bool
+    {
+        $currentVersion = $this->getVersion();
+
+        // Aurora is managed by AWS and is updated automatically if EOL, therefor we can ignore that here
+        if (str_contains(strtolower($currentVersion), 'aurora')) {
```

Comment:
This check will not work, at least not with a current 8.x-aurora:

```
mysql> SELECT VERSION();
+-----------+
| VERSION() |
+-----------+
| 8.0.39    |
+-----------+
1 row in set (0.00 sec)

mysql> SELECT @@VERSION;
+-----------+
| @@VERSION |
+-----------+
| 8.0.39    |
+-----------+
1 row in set (0.00 sec)
```

I think we can only check for variables like `@@aurora_version`, but the plain `VERSION()` seems to no longer report any "aurora" hint.

Reply:
just tried selecting both aurora_version() and @@aurora_version on standard mysql and it fails with an error, so can't even select both and decide what to use. Can we assume cloud would always be on a supported version? or have other mechanism to suppress the warning on cloud regardless of the actual db used.

Reply:
changed that.